src/domains/messages-common/05_apim_v2.tf

```
@@ -188,8 +188,60 @@ data "azurerm_api_management_product" "apim_v2_product_services" {
   resource_group_name = data.azurerm_api_management.apim_v2_api.resource_group_name
 }
 
+# APIM APIs
+
+data "http" "messages_sending_external_openapi" {
+  url = "https://raw.githubusercontent.com/pagopa/io-functions-services-messages/master/openapi/index_external.yaml"
+}
+
+module "apim_v2_messages_sending_external_api_v1" {
+  source = "git::https://github.com/pagopa/terraform-azurerm-v3//api_management_api?ref=v7.69.1"
```

Hello! I would suggest to upgrade the module version to the latest if no breaking changes are found during plan

---

```suggestion
  source = "git::https://github.com/pagopa/terraform-azurerm-v3//api_management_api?ref=v8.11.0"
```
The latest version currently is the 8.11.0

---

Is it 8.11.0?